src/lib.rs

```diff
@@ -463,12 +463,21 @@ fn execute_instr(input: InstrContext) -> Option<InstrEffects> {
 
     let clock = match sysvar_cache.get_clock() {
         Ok(clock) => (*clock).clone(),
-        Err(_) => Clock::default(),
+        Err(_) => {
+            let mut default_clock = Clock::default();
+            default_clock.slot = 10;
```

Comment:
Where is 10 coming from? Could you leave a comment with a link to the Solana source?